### ISSUE TITLE
HDF5 files cannot be merged with the latest lstchain

### ISSUE THREAD

**Issue by SeiyaNozaki** [cta-observatory/cta-lstchain#172]
Hello.
I cannot merge hdf5 files by "scripts/merge_hdf5_files.py" with the latest version of cta-lstchain. It seems it is caused from the recent commit (#149 ). Disp parameters are concatenated after the HDF5TableWriter for 'LSTCam/events', then nodes of this container cannot be found and merge was not done.
I tried to fix it, but I'm not sure what is the best solution...
Could you please check it?

**Comment by rlopezcoto** [comment]
Hi @SeiyaNozaki,
thanks for reporting this. Could you please commit your code so we can have a look? We really appreciate fixes, so do not be shy about having it committed :)

**Comment by SeiyaNozaki** [comment]
I tried to fix it, but I didn't get how to solve it, then gave up... :(

What I checked is just to print the nodes and children.

```Swift
    for filename in file_list[1:]:
        with tables.open_file(filename) as file:
            try:
                for ii, node in nodes.items():
                    print(node)
                    for children in node:
                        print(children)
                        p = os.path.join(ii, children.name)
                        node[children.name].append(file.root[p].read())
            except:
                print("Can't merge {}".format(filename))
```

Then, I got the previous output with the old DL1 data.
```
(cta-dev) [cta@dragon6 scripts]$ python merge_hdf5_files.py -d ~/Work/CTA/MCdata/proton_diffuse_dl1_20190828/temp/
/home/cta/Work/Install/anaconda3/envs/cta-dev/lib/python3.7/site-packages/sklearn/externals/joblib/__init__.py:15: DeprecationWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=DeprecationWarning)
/events (Group) ''
/events/LSTCam (Table(223,)) 'Storage of DL1ParametersContainer'
/simulation (Group) ''
/simulation/run_config (Table(1,)) 'Storage of MCHeaderContainer'
Closing remaining open files:merge.h5...done/home/cta/Work/CTA/MCdata/proton_diffuse_dl1_20190828/temp//dl1_proton_20deg_0deg_run100___cta-prod3-demo-2147m-LaPalma-baseline-mono.simtel.h5...done
```

But now I got like below.
```
(cta-dev) [cta@dragon6 scripts]$ python merge_hdf5_files.py -d ~/Work/CTA/ctaana/MonoAnalysis/temp/
/home/cta/Work/Install/anaconda3/envs/cta-dev/lib/python3.7/site-packages/sklearn/externals/joblib/__init__.py:15: DeprecationWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=DeprecationWarning)
/events (Group) ''
/events/LSTCam (Group) ''
Can't merge /home/cta/Work/CTA/ctaana/MonoAnalysis/temp//dl1_proton_20deg_0deg_run101___cta-prod3-demo-2147m-LaPalma-baseline-mono.simtel.h5
Closing remaining open files:/home/cta/Work/CTA/ctaana/MonoAnalysis/temp//dl1_proton_20deg_0deg_run100___cta-prod3-demo-2147m-LaPalma-baseline-mono.simtel.h5...donemerge.h5...done
```
So the output of "children" is "Group" instead of "Table" with the latest version of lstchain.

**Comment by vuillaut** [comment]
Hi @SeiyaNozaki 

Thank you for reporting this bug!
And sorry for the inconvenience.

I think this bug is fixed with #120.

**Comment by vuillaut** [comment]
I think this is fixed.
